gen3users/user_security.py

```diff
@@ -184,6 +208,21 @@ def _receive_input(anything_found):
     return
 
 
+def _f(subrsrc, phsid, locs):
```

Comment:
It's a little unclear what this method is trying to do. Might need some name adjustments and maybe a comment.